Regression/prepare_file_travis.py

```diff
@@ -129,6 +129,7 @@ def select_tests(blocks, match_string_list, do_test):
 
 if ci_python_main:
     test_blocks = select_tests(test_blocks, ['PYTHON_MAIN=TRUE'], True)
+    test_blocks = select_tests(test_blocks, ['USE_EB=TRUE'], False)
```

Comment:
ok, but mid-term I would say we should try to make `USE_EB` on by default so we can cover it well.
(And only adding runtime overhead/data if EBs are used from inputs)